ui/ozone/platform/wayland/wayland_connection.cc

```diff
@@ -489,7 +514,7 @@ void WaylandConnection::Global(void* data,
   } else if (!connection->data_device_manager_ &&
              strcmp(interface, "wl_data_device_manager") == 0) {
     wl::Object<wl_data_device_manager> data_device_manager =
-        wl::Bind<wl_data_device_manager>(registry, name, 1);
+        wl::Bind<wl_data_device_manager>(registry, name, version);
```

Comment:
You mustn't pass the version back. Instead, create a constexpr uint32_t kVersion = "your version" and test it against passed version. If the version is suitable, create the object.